### ISSUE TITLE
Link to hgnc.owl?

### ISSUE THREAD

**Issue by bherr2** [musen-lab/hgnc2owl#1]
Can you publish and / or link to the final hgnc.owl? This is a great contribution!

**Comment by bherr2** [comment]
Ooops, I see the .gz now. Nevermind!

**Comment by bherr2** [comment]
Oh but actually, is this published to bioportal / etc?

**Comment by johardi** [comment]
Yes, the file is also available in BioPortal: https://bioportal.bioontology.org/ontologies/HGNC-NR
